src/app/components/ScrollingDataDisplay/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import { COLORS } from '../../../styles/theme/colors'
+
+export const ScrollingDataDisplay: FC<{ data: string; fontWeight?: number }> = ({
+  data,
+  fontWeight = 700,
+}) => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        padding: '10px 0px 0px 0px',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        gap: '10px',
+      }}
+    >
+      <Box
+        sx={{
+          borderRadius: '5px',
+          background: COLORS.grayLight,
+          border: `1px solid ${COLORS.grayMedium}`,
+          // maxHeight: '200px',
+          height: '349px',
+          overflowY: 'scroll',
+          padding: '10px',
+        }}
+      >
+        <Typography
```

Comment:
Can this be replaced by `LongDataDisplay`?

Reply:
I don't think so, because it is designed for collapsing / expanding, while here we want a fixed height with scrolling.